config/locales/lv.yml

```diff
@@ -9,4 +9,22 @@ lv:
   archived_articles: "Arhivētie raksti"
   new_article: "Jauns raksts"
   edit: "Rediģēt"
-  delete: "Dzēst"
\ No newline at end of file
+  delete: "Dzēst"
+  home: "Sākums"
+  commenter: "Komentētājs"
+  comment: "Komentārs"
```

Comment:
Pārkārto tulkojumus, lai modeļu un atribūtu tulkojumi būtu pieejami atbilstoši šim https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models
Tas pats priekš tulkojumiem iekš viewiem